### ISSUE TITLE
Request for Dataset Split Details

### ISSUE THREAD

**Issue by Quarkstar** [Ablustrund/APPS_Plus#5]
Thank you for your amazing work on this project!

I was wondering if you could provide some additional details about the dataset split. I understand that you mentioned the dataset was split randomly, but I'd love to get the exact index of the test set and validation set if possible. This would be super helpful for me to reproduce your results and build upon your work.

Thank you again for your time and effort.

**Comment by Ablustrund** [comment]
Sure! We will release the new version of the APPS+, including more unit tests and the index of the training set, val set, and test set.

**Comment by Ablustrund** [comment]
In two days!

**Comment by Ablustrund** [comment]
Hi, we clean the function name of the canonical solution in each programming problem. The final version of our APPS+ dataset is available at here: https://drive.usercontent.google.com/download?id=1dTRJHXhMcPz4CPt73f2vju5KdLmqcP4F&export=download&authuser=0

In addition, the samples for validation and testing are both in the company's laptop, and we will give you their index in two days.

**Comment by Ablustrund** [comment]
The test set is: [Uploading cut.json…]()

**Comment by Ablustrund** [comment]
[cut.json](https://github.com/user-attachments/files/17175448/cut.json)
